app/services/network_cache/fetcher.rb

```diff
@@ -8,10 +8,12 @@ def self.from(testing_ground, opts = {})
 
     #
     # Fetches all cache and sets is as the load attribute for a node
-    def fetch
+    def fetch(nodes = [])
       tree_scope.each do |network|
         network.nodes.each do |node|
```

Comment:
This is something which I have put in an [issue](https://github.com/quintel/etmoses/issues/722) for later. The thing is that the `TreeSampler` also contains a loop over all nodes which might than be simplified. And what to do when the nodes are empty? I'll put it in an issue and I'll figure this out later on. Thanks for this comment.